controllers/categories.controllers.js

```diff
@@ -0,0 +1,11 @@
+const {fetchAllCategories} = require('../models/categories.model')
+
+
+function getAllCategories(req, res, next) {
+    fetchAllCategories()
+    .then((categories) => {
+        res.status(200).send({categories})
+    })
```

Comment:
Although very little could go wrong we should still have a catch block on the controller.